src/User.java

```diff
@@ -2,14 +2,13 @@
 import java.util.List;
 
 public class User {
-
- static List<String> infoUser = new ArrayList<>();
     String name;
     String surname;
+    int id;
     String login;
     String password;
 
-    public void addInfo(String name, String surname, String login, String password) {
+    public void addUser(String name, String surname, String login, String password) {
```

Comment:
user.addUser() ? what does it mean?
if you want to add data to user you have 2 options:
- Add this data via constructor
- Use setters